centreon/www/install/php/Update-2.8.24.php

```diff
@@ -43,7 +43,7 @@
     '`service_id` int(11) DEFAULT NULL, ' .
     'UNIQUE KEY (`group_id`,`host_id`,`service_id`), ' .
     'KEY `index1` (`host_id`,`service_id`,`group_id`) ' .
-    ') ENGINE=InnoDB DEFAULT CHARSET=utf8 ';
```

Comment:
IMHO : Previous upgrade scripts should not be updated. The utf8 tables modifications should be added in the 23.04.0 script